docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -364,10 +364,6 @@ In most cases, a Bareos major upgrade can be achieve by:
 It is generally sufficient to upgrade directly to the latest release, without having to install any intermediate releases.
 However, it is required to read the release notes of all intermediate releases.
 
-One exception is when using a |mysql| Bareos catalog,
-which have been removed with Bareos :sinceVersion:`21.0.0: MySQL backend removed`.
-Therefore you first have to upgrade to Bareos 20 and migrate the |mysql| into a |postgresql| Bareos Catalog, see :ref:`section-MigrationMysqlToPostgresql`.
-
```

Comment:
see my comment above, we should probably still mention that you may need to do this. Some people may still want upgrade from Bareos 16.2 (or something like that) to 22.